### ISSUE TITLE
Add installing with winget section to Readme.md and homepage

### ISSUE THREAD

**Issue by dinhanhx** [biolab/orange3#4933]
Like the title.
Recently, my [pull request in winget-pkgs (Microsoft)](https://github.com/microsoft/winget-pkgs/pull/2898) has been successfully merged, which means now you can use [winget](https://github.com/microsoft/winget-cli) to install Orange on Windows.

Cmd:
## To search
```
winget search Orange
```
Output
```
C:\WINDOWS\system32>winget search Orange
Name      Id                           Version Match
---------------------------------------------------------------
Orange    UniversityofLjubljana.Orange 3.26.0
```

## To install
```
winget install Orange
```
or with id
```
winget install --id  UniversityofLjubljana.Orange
```

**Comment by ajdapretnar** [comment]
Would you mind creating a PR with the modified README?

**Comment by dinhanhx** [comment]
Okay I will, wait for few minutes

**Comment by dinhanhx** [comment]
@ajdapretnar I just created the PR. It's #4935

**Comment by thocevar** [comment]
Closed by #4935
